package.json

```diff
@@ -26,7 +26,6 @@
         "changeset-publish": "yarn prepublishOnly && yarn changeset version && yarn install && yarn lint:versions --fix && yarn update-version && yarn changeset publish --no-git-tag && yarn push-to-remote && yarn create-git-tag && yarn postpublish",
         "update-version": "node ./tasks/update-version.js",
         "chromatic": "chromatic --build-script-name storybook:build # note that --project-token must be set in your env variables",
-        "create-git-tag": "node --no-warnings tasks/create-git-tag.js",
```

Comment:
I don't see this used or documented anywhere so is it safe to clean this up? We don't generate tags manually anyway now that we're using changesets.